annotations/views/relationtemplate_views.py

```diff
@@ -50,6 +50,22 @@ def list(self, request):
         result = serializer.data
         return Response(result)
 
+    @action(detail=True, methods=['post'])
+    def create_relation(self, request, pk=None):
+        template = get_object_or_404(RelationTemplate, pk=pk)
+        data = request.data
+        text = get_object_or_404(Text, pk=data['occursIn'])
+        
+        project_id = data.get('project', None)
+        if project_id is None:
+            project_id = VogonUserDefaultProject.objects.get(
+                for_user=request.user).project.id
+        
+        relationset = relations.create_relationset(
+            template, data, request.user, text, project_id
+        )
+        return JsonResponse({ 'relationset': relationset.id })
```

Comment:
Lets change the name to 'relationset_id' so that its clear that its just the id and not an object.

Reply:
Yeah. Made the change